pydtk/bin/sub_commands/model.py

```diff
@@ -131,7 +131,7 @@ def generate(
             data['record_id'] = record_id
 
         # Display
-        print(json.dumps(data, indent=4, default=_default_json_handler))
+        return json.dumps(data, indent=4, default=_default_json_handler)
```

Comment:
For https://github.com/dataware-tools/pydtk/pull/127/files#r1043080325

Reply:
revert https://github.com/dataware-tools/pydtk/pull/127/commits/83c35c4481112b2a0d86bc632a73c14bf481e2a8